src/mmw/apps/home/views.py

```diff
@@ -42,9 +44,12 @@ def model(request, proj_id=None):
     csrf_token.update(csrf(request))
 
     if proj_id:
-        get_object_or_404(Project, user=request.user.id, id=proj_id)
+        project = get_object_or_404(Project, id=proj_id)
 
-    return render_to_response('home/home.html', csrf_token)
+    if project.user.id == request.user.id or project.is_private is False:
```

Comment:
I'm new to Python and its idioms so I might be wrong, but would `or not project.is_private` be a better expression than `or project.is_private is False`?